src/containers/BooksForm.js

```diff
@@ -23,13 +23,13 @@ const BooksForm = ({ createBook }) => {
 
   return (
     <div>
-      <h2>Create a book</h2>
-      <form onSubmit={event => handleSubmit(event)}>
+      <h2 className="new-book-head">Add a new book</h2>
+      <form className="display-grid" onSubmit={event => handleSubmit(event)}>
```

Comment:
- [ ] **[OPTIONAL]** The form used to create a book should have validations, you should not be able to create an empty book. We can add a `validate` function before creating the book and maybe display errors in the form to inform the user.

Reply:
I didn't implement that but I used the `required` flag on input field,